akvo/rest/views/report.py

```diff
@@ -52,6 +52,45 @@ def report_formats(request):
     })
 
 
+@api_view(['GET'])
+def organisation_reports(request):
+    queryset = Report.objects.prefetch_related('organisations', 'formats')\
+        .filter(url__icontains='{organisation}')\
+        .exclude(url__icontains='program=true')
```

Comment:
Why do we need this kind of a URL? Why can't we have `{program}` for program reports?

Reply:
It's just for convenience, at least for now, because previously the reports are an organisation report, and it won't be easy to update the existing reports that still lives in ReportServer. But once we have migrated all the reports to be generated by python then it will be more doable.

Reply:
How many such reports do we have, which were previously organisation reports and now will be program reports?

Reply:
Currently 8 reports.